dts/bindings/device_node.yaml.template

```diff
@@ -1,88 +1,103 @@
-title: <short description of the node>
+title: Short description of the node
 
 description: >
-    Longer free-form description of the node, with spanning
-    lines
+    Longer free-form description of the node.
+    Can go over multiple lines.
 
+# Bindings are often based on other bindings, which are given in 'inherits'.
+# The resulting binding is the union of the inherited bindings and this binding
+# (internally, it's a recursive dictionary merge).
+#
+# If a field appears both in this binding and in a binding it inherits, then
+# the value in this binding takes precedence. This can be used to change a
+# 'required: false' from an inherited binding to a 'required: true' (see the
+# 'properties' description below).
 inherits:
-  - !include other.yaml # or [other1.yaml, other2.yaml]
-# Files with other bindings that also apply to the node. If an attribute is set
-# both in an included file and in the file that includes it, then the value
-# from the including file (the file with the !include) is used.
+    !include other.yaml # or [other1.yaml, other2.yaml]
 
-< parent | child >:
-# parent/child is used to document implicit relation between nodes.
-# This information is required to generate parent related bits in child
-# attributes.
-# In case parent has 'bus', slave inherits some information from master.
-# parent and child should share same bus-type value.
-   bus: <bus-type>
+# If the node describes a bus, then the bus type should be given, like below
+parent:
+    bus: <string describing bus type, e.g. "i2c">
 
-sub-node:
-# Used for cases in which a dts node has children, and the children dont
-# require/specify a 'compatible' property.  The sub-node is effective the
-# binding for the child.
+# If the node appears on a bus, then the bus type should be given, like below.
 #
-# Here's an example for a pwm-leds binding in which the child nodes
-# would be required to have 'pwms' properties.
+# When looking for a binding for a node, the code checks if the binding for the
+# parent node contains 'parent: bus: <bus type>'. If it does, then only
+# bindings with a matching 'child: bus: <bus type>' are considered. This allows
+# the same type of device to have different bindings depending on what bus it
+# appears on.
+child:
+    bus: <string describing bus type, e.g. "i2c">
 
+# Used to simplify cases where a node has children that can all use the same
```

Comment:
"Sub-nodes are used to ..." would make it clearer.

Reply:
Fixed